PlayerController.cs

```diff
@@ -1,36 +1,38 @@
 // TODO: Problem 4: Write in the comments what each line of code is doing
-using System.Collections;
+using System.Collections; //phrase "using" tells script where to get base functions
 using System.Collections.Generic;
 using UnityEngine;
 
 public class PlayerController : MonoBehaviour
 {
     // TODO: Problem 1: Define a private variable for your player or character's name and print it out to Unity's console
+    private string username; //instantiates a private variable
     // Start is called before the first frame update
     void Start()
     {
-        
+        username = "WHATTSUPPP"; //the string variable is defined
+        Debug.Log("Your character is called " + username); //unity's way of print()
     }
 
     // Update is called once per frame
     void Update()
     {
-        
+        //pass
     }
 
-    [Header("General Setup Settings")]
-    [SerializeField] private InputAction movement;
-    [Tooltip("How fast player moves up and down based upon player input")][SerializeField] float controlSpeed = 30f;
+    [Header("General Setup Settings")] //ui
```

Comment:
These are called [Unity Attributes](https://docs.unity3d.com/Manual/Attributes.html). In a way, it is technically UI where it is a new line of interface for Unity's Inspector Editor. Usually, we refer to UI for Unity's Inspector to be used with UIToolkit and UI for games would be used with UIToolkit and uGUI.